modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -744,7 +744,7 @@ public class ApiClient {
      */
     public boolean isJsonMime(String mime) {
       String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
-      return mime != null && mime.matches(jsonMime);
+      return mime != null && (mime.matches(jsonMime) || mime.equals("application/json-patch+json"));
```

Comment:
It would be nice to add a test case too: https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/java/okhttp-gson/src/test/java/io/swagger/client/ApiClientTest.java#L81